creating_citation_counts_tsv/scripts/summarize_gene_citations_all_species.py

```diff
@@ -0,0 +1,335 @@
+""" 
+Creating TSVs containing the 10 most-cited genes per species and their gene information.
+"""
+import os
+import sys
+import json
+import pyspark
+import requests
+from operator import add
+from datetime import date, timedelta, datetime
+import csv
+from collections import Counter
+
+
+def getting_recent_gene2pubmed(species):
```

Comment:
Functions typically use names like `get` instead of `getting`.  For convention:

```suggestion
def get_recent_gene2pubmed(species):
```

And similarly elsewhere (i.e. remove "ing" from all function definitions and calls).